### PR TITLE
Allow support for single file upload

### PR DESCRIPTION
**Pull request by tiarly**
It closes https://github.com/refile/refile/issues/348

### CODE REVIEW

**Comment by tiarly** [comment]
The failing CI build isn't really related to this pull request code 😉

File: README.md [hunk 1]
```diff
@@ -784,6 +785,62 @@ class Post < ActiveRecord::Base
...
+attribute:
+
+``` erb
+<%= form_for @post do |form| %>
+  <%= form.label :images_files %>
+  <%= form.attachment_field :images_files %>
+<% end %>
+```
+
+Now you only need to permit the generated accessor in your controller.  Since
```
Comment (abnersajr) [inline README.md]:
```suggestion
Now you only need to permit the generated accessor in your controller. Since
```

**Comment by sobrinho** [comment]
@tiarly can you open another PR fixing that build problem?

I can't merge a broken build, even when it is not the root cause.

**Comment by tiarly** [comment]
> @tiarly can you open another PR fixing that build problem?
> 
> I can't merge a broken build, even when it is not the root cause.

Will do!

**Comment by sobrinho** [comment]
Thanks! <3

**Pull request merged**